packages/utils/src/internals/robots.ts

```diff
@@ -0,0 +1,58 @@
+import { downloadListOfUrls, gotScraping } from '@crawlee/utils';
+import { HTTPError } from 'got';
```

Comment:
You cannot use this due to got and got-scraping being ESM only. 

you'll need to do
```ts
// @ts-expect-error This throws a compilation error due to got-scraping being ESM only but we only import types, so its alllll gooooood
import type { HTTPError as HTTPErrorClass } from 'got';

let HTTPError: typeof HTTPErrorClass;
```

and then in your catch block, 
```ts
if (!HTTPError) {
   ({ HTTPError } = await import('got'));
}
```